app/config/collections.php

```diff
@@ -2233,6 +2394,20 @@
                 'lengths' => [],
                 'orders' => [Database::ORDER_ASC],
             ],
+            [
+                '$id' => ID::custom('_key_installationId'),
```

Comment:
Is this good enough? Should we also create a shared index?
@vermakhushboo could be great to verify and optimize together with @fogelito.